__tests__/app.test.js

```diff
@@ -89,3 +90,32 @@ describe("GET /api/article/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: Responds with an array of article objects sorted by date in descending order, each containing a comment count", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(13);
+        articles.forEach((article) => {
+          expect(article).toEqual({
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            article_id: expect.any(Number),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+          expect(article).not.toHaveProperty("body");
+        });
+        expect(articles).toBeSorted({
```

Comment:
as we're testing a different behaviour here, we should have a separate test for this :)